pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
I can guarantee this is your problem, I have the same issue regularly.

Reply:
It worked ! Thanks, I wonder what's the underlying issue

Reply:
who knows :shrug:

Reply:
Fixed by #463